### PR TITLE
add new style to push report table containing wells out wide with the tables

### PR DESCRIPTION
**Pull request by jtreuting**
a new class is needed since it doesn't work well with all the well's (yes pun was intended!)

**Before**
wide-tables-before

**After**
wide-tables-after

### CODE REVIEW

File: webapp/src/main/webapp/src/styles.less [hunk 1]
```diff
@@ -496,3 +496,10 @@ body .select-menu-group {
   white-space: nowrap;
   padding: .2em 0 .3em;
 }
+
+/** used on well's containing wide data tables to push the well out with the table */
+.report-table-container {
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/styles.less]: can we make this not specific to report tables? The reason I ask is I am doing something similar for the aggregate reports

Comment (jtreuting) [reply webapp/src/main/webapp/src/styles.less]: sure, i'm not sure a better more generic name.  maybe, wide-content-container?

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/styles.less]: that sounds good 👍

File: webapp/src/main/webapp/src/styles.less [hunk 2]
```diff
@@ -496,3 +496,10 @@ body .select-menu-group {
   white-space: nowrap;
   padding: .2em 0 .3em;
 }
+
+/** used on well's containing wide data tables to push the well out with the table */
+.wide-content-container {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: I'm thinking we'll probably want all wells to behave this way in the app no? Eventually we might want to extend the bootstrap `.well` class in the ui kit. @aaronsleeper thoughts?

File: webapp/src/main/webapp/src/styles.less (hunk 1 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/styles.less]: This would have been my name of choice:

**Pull request merged**